deploy/helm/prometheus-overrides.yaml

```diff
@@ -75,8 +75,6 @@ prometheusOperator:
   prometheus-node-exporter:
     ## Additional labels for pods in the DaemonSet
     podLabels: {}
-    ## Additional annotations for pods in the DaemonSet
-    podAnnotations: {}
```

Comment:
This is supported in the node-exporter dependency like you have above - https://github.com/helm/charts/blob/master/stable/prometheus-node-exporter/templates/daemonset.yaml#L21